ADOPTERS.md

```diff
@@ -92,6 +92,8 @@ production users that have added themselves (in alphabetical order):
   and identify high-risk configurations. The policies ingest 1,000s of
   AWS resources to generate the final report.
 
+* [Mercari](https://www.mercari.com/) uses OPA to enforce admission control policies in their multi-tenant Kubernetes clusters. It helps maintain the governance of the cluster, checking that developers are following the best practices in the admission controller. They also use the [confest](https://github.com/open-policy-agent/conftest) to enforce policies in their CI/CD pipeline.
```

Comment:
I think I'd skip the "the" in front of "conftest", so that "the conftest" -> "conftest".

Reply:
Thank you for your kind review.
I fixed the inline and and removed the `the` in this commit 👉 https://github.com/open-policy-agent/opa/pull/3572/commits/bcb9e4bf3afaa78e3b4e90db7a34587fa0fe2655

Reply:
Thanks! If you can squash the two commits into one I'll merge this right away :)